addons/mail/static/src/components/emoji_grid_loading_screen/emoji_grid_loading_screen.scss

```diff
@@ -0,0 +1,7 @@
+.o_EmojiGridLoadingScreen {
+}
+
+.o_EmojiGridLoadingScreen_spinner {
+    filter: grayscale(1);
```

Comment:
I don't see any visual difference with or without grayscale. The icon is already a greyish.